src/parser.js

```diff
@@ -312,6 +317,10 @@ export default class Parser extends Stream {
         },
         comment() {
           // comments are not important for playback
+          if (!currentUri.tags) {
+            currentUri.tags = [];
+          }
+          currentUri.tags.push(entry.line);
```

Comment:
I don't think we need annotations for tags that we're already parsing correctly.